db/db_impl/db_impl.h

```diff
@@ -2283,6 +2282,13 @@ class DBImpl : public DB {
   Directories directories_;
 
   WriteBufferManager* write_buffer_manager_;
+  // For simplicity, CF based write buffer manager does not support stall the
+  // write.
+  // Note: std::shared_ptr<WriteBufferManager> is store in ColumnfamilyOptions
+  // which is destroyed before DBimpl, so we use
```

Comment:
Have you had any actual error caused by not using `shared_ptr`? All code using `WriteBufferManager` should be destroyed before `VersionSet`, there shouldn't be any problem.